en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+* Once the issue is confirmed, create a _draft security advisory_ against the relevant GitHub repository. This will allow you to request a CVE.
+* Once a CVE has been assigned, respond to issue reporter and add it to the Github issue.
+* Ask the reporter if they want to be credited for the disclosure and under what name.
+* Clarify who picks up and owns the issue (assign in Github).
+  The owner can be separate from the developer resolving the issue,
+  their primary responsibility is to ensure the issue keeps moving through the process correctly.
+
+### When developing a fix
+
+* Ensure you're working on the oldest supported minor release branch of every supported major release (see [Supported Versions](#supported-versions))
+* Move the issue into "In Progress" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+* Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo. Don't forget to update branches from the upstream repo.
+* Ensure that all security commit messages are prefixed with the CVE. E.g. "[CVE-2019-001] Fixed invalid XSS"
+
+### Before release
+
+* For issues rated "high" or "critical" (CVSS of >=7.0), post a pre-announcement to the [security pre-announcement list](https://groups.google.com/a/silverstripe.com/forum/#!forum/security-preannounce).
+  It should include a basic "preannouncement description" which doesn't give away too much,
+  the CVSS score as well as the CVE identifier.
+* Create a draft page under [Open Source > Download > Security Releases](https://www.silverstripe.org/admin/pages/edit/show/794).
+  Populate it with the information from the [Github project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Link to silverstripe.org security release page in the changelog.
+* Update the _draft security advisory_ on GitHub.
+* Move the issue to "Awaiting Release" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+ 
+### After release
+
+* Publish silverstripe.org security release page
+* Respond to issue reporter with reference to the release on the same discussion thread (cc security@silverstripe.org)
+* Publish the security advisory on GitHub. This will also publish the CVE.
+* Open a pull request to [https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe](https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe) repository.
+* Move the issue to "Done" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+
+### Detailed CVE and CVSS Guidance
+
+* In the GitHub _draft security advisory_, we follow certain conventions on fields:
+  * **Affected product** should have one entry for each composer package affected by the vulnerability. If the patch is backported to older release lines, there should be entries for those older releases.
```

Comment:
Define "affected" in this context. Does it mean for the module that holds the vulnerable code and all modules which have it as a dependency?

Reply:
The GitHub draft advisory form seems pretty self-explanatory to me. It's not clear to me that we need much more detail than that.

Maybe have a look and let me know if you disagree.

Reply:
I don't have access to actually see the form, but looking at [the docs](https://docs.github.com/en/code-security/repository-security-advisories/creating-a-repository-security-advisory) it's not immediately clear to me whether it means _just_ the module(s) that holds the vulnerable code directly, or if it also means all modules which have it as a dependency.

Reply:
This part hasn't been clarified?